src/plugins/data/common/search/es_search/types.ts

```diff
@@ -24,6 +24,13 @@ export const ES_SEARCH_STRATEGY = 'es';
 export interface IEsSearchRequest extends IKibanaSearchRequest {
   params: SearchParams;
   indexType?: string;
+
+  /*
+    An identifier used to mark multiple search requests as originating from a single
+    application session. In particular, this argument is used to restore long running
```

Comment:
nit: Might make the concept clearer by providing a concrete example, e.g. "from a single application session, such as all of the search requests initiated by a dashboard."